frontend/apps/reader/modules/readerlink.lua

```diff
@@ -61,6 +61,10 @@ local function swipe_to_go_back()
     return G_reader_settings:readSetting("swipe_to_go_back") == true
 end
 
+local function swipe_to_follow_first_link()
```

Comment:
function name should be camelCase. There are some legacy code that's not updated to match the style guide.